app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -32,11 +32,14 @@ def description
     ManageIQ::Providers::Azure::Regions.find_by_name(provider_region)[:description]
   end
 
-  def self.raw_connect(clientid, clientkey, azuretenantid)
+  def self.raw_connect(clientid, clientkey, azuretenantid, proxy_uri = nil)
```

Comment:
Can we change the parameter names to be standard snake case for better readability?
`clientid` to `client_id`
`clientkey` to `client_key`
`azuretenantid` to `azure_tenant_id`